components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2439,26 +2451,27 @@ bool RemoveRecurringTipOnFileTaskRunner(
   return backend->RemoveRecurringTip(publisher_key);
 }
 
-void RewardsServiceImpl::OnRemovedRecurringTip(
-    ledger::RecurringRemoveCallback callback, bool success) {
+void RewardsServiceImpl::OnRemoveRecurringTip(
+    ledger::RemoveRecurringTipCallback callback,
+    bool success) {
```

Comment:
Should we be sending back a `LedgerResult` instead of `bool`?

Reply:
db returns bool, so let's for now stay with that